doc/Makefile

```diff
@@ -31,7 +31,7 @@ help:
 # pdf: .install-deps
 pdf:
 	@$(SPHINXBUILD) -M latex "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-	cd $(BUILDDIR)/latex && latexmk -r latexmkrc -pdf *.tex -interaction=nonstopmode || true
+	cd $(BUILDDIR)/latex && latexmk -r latexmkrc -pdf -xelatex *.tex -interaction=nonstopmode || true

```

Comment:
Why not do the touch here in that case? Leave it with a TODO if you prefer to fix it in the future